sdv/constraints/tabular.py

```diff
@@ -539,6 +541,158 @@ def reverse_transform(self, table_data):
         return table_data
 
 
+class Between(Constraint):
+    """Ensure that the ``constraint_column`` is always between ``high`` and ``low``.
+
+    The transformation strategy works by replacing the ``constraint_column`` with a
+    scaled version and then applying a logit function. The reverse transform
+    applies a sigmoid to the data and then scales it back to the original space.
+
+    Args:
+        constraint_column (str):
+            Name of the column to which the constraint will be applied.
+        low (float or str):
+            If float, lower bound on the values of the ``constraint_column``.
+            If string, name of the column which will be the lower bound.
+        high (float or str):
+            If float, upper bound on the values of the ``constraint_column``.
+            If string, name of the column which will be the upper bound.
+        strict (bool):
+            Whether the comparison of the values should be strict ``>=`` or
+            not ``>`` when comparing them. Currently, this is only respected
+            if ``reject_sampling`` or ``all`` handling strategies are used.
+        handling_strategy (str):
+            How this Constraint should be handled, which can be ``transform``
+            or ``reject_sampling``. Defaults to ``transform``.
+    """
+
+    def __init__(self, column, low, high, strict=False, handling_strategy='transform',
```

Comment:
We should also add `low_is_scalar` and `high_is_scalar` arguments like in the `GreaterThan` constraint.